subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -1111,29 +1120,51 @@ static void set_bn_max_pdu(bool framed, uint32_t iso_interval,
 			   uint8_t *max_pdu)
 {
 	if (framed) {
-		uint32_t ceil_f_x_max_sdu;
-		uint16_t max_pdu_bn1;
-		uint32_t max_drift;
+		uint32_t max_drift_us;
 		uint32_t ceil_f;
 
-		/* Framed (From ES-18002):
+		/* BT Core 5.4 Vol 6, Part G, Section 2.2:
 		 *   Max_PDU >= ((ceil(F) x 5 + ceil(F x Max_SDU)) / BN) + 2
 		 *   F = (1 + MaxDrift) x ISO_Interval / SDU_Interval
 		 *   SegmentationHeader + TimeOffset = 5 bytes
 		 *   Continuation header = 2 bytes
 		 *   MaxDrift (Max. allowed SDU delivery timing drift) = 100 ppm
 		 */
-		max_drift = DIV_ROUND_UP(SDU_MAX_DRIFT_PPM * sdu_interval, 1000000U);
-		ceil_f = DIV_ROUND_UP(iso_interval + max_drift, sdu_interval);
-		ceil_f_x_max_sdu = DIV_ROUND_UP(max_sdu * (iso_interval + max_drift),
-						    sdu_interval);
-
-		/* Strategy: Keep lowest possible BN.
-		 * TODO: Implement other strategies, possibly as policies.
+		max_drift_us = DIV_ROUND_UP(SDU_MAX_DRIFT_PPM * sdu_interval, 1000000U);
```

Comment:
Plesae do the below in all occurrences.

```suggestion
		max_drift_us = DIV_ROUND_UP(SDU_MAX_DRIFT_PPM * sdu_interval, USEC_PER_SEC);
```